index.html

```diff
@@ -49,5 +49,167 @@ <h1 class="headline-title">Hey there. I'm Rachel</h1>
+                                    <li>css</li>
+                                    <li>javascript</li>
+                                    <li>html</li>
+                                </ul>
+                                <div><button type="button" class="project-btn">See Project</button></div>
+                            </div>
+                            </article>
+                            <article>
+                                <div class="works-card">
+                                    <div class="works-card-image">
+                                        <img src="./img/Img Plaholder.svg" alt="">
+                                    </div>
+                                    <h3 class="works-card-title">Multi-Post Stories Gain+Glory</h3>
+                                    <ul class="works-card-skills">
+                                        <li>ruby on rails</li>
+                                        <li>css</li>
+                                        <li>javascript</li>
+                                        <li>html</li>
+                                    </ul>
+                                    <div><button type="button" class="project-btn">See Project</button></div>
+                                </div>
+                                </article>
+                            </div>
+                        </section>
+                        <!--ABOUT SECTION-->
+                        <section class="container-flex about-me">
+                                <div class="container">
+                                    <h2>About Me</h2>
+                                    <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                                        don't hestiate to contact me.</p>
+                                    <button type="button" class="resume-btn">Get My Resume</button>
+                                </div>
+                            </div>
+                            <ul class="about-section-skills">
+                                <li>
+                                <article class="about-card">
+                                        <div class="about-card-image">
+                                           <img src="./img/Icon Langguage.svg" alt="language"> 
+                                        </div>
+                                        <h2>Languages</h2>
+                                        <ul class="about-card-skills">
+                                            <li>Javascript</li>
+                                            <li>Ruby</li>
+                                            <li>Html</li>
+                                            <li>Css</li>
+                                        </ul>
+                                    </div>
+                                </article>
```

Comment:
- [ ] as the previous reviewer requested try to follow Figma as much as possible :

1. kindly reduce the space between the icon and the title  
2. kindly add the white background circle for the icon
3. please fix the layout of the about-card-skills layout

Figma          | Your Design
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/89282221/192094663-22ae18a8-9411-4e32-8ebf-83d5d1120cd5.png) | ![image](https://user-images.githubusercontent.com/89282221/192094549-e7d7725e-9087-40a4-a37f-edc1d72ca3d1.png)